spec/mailers/comment_mailer_spec.rb

```diff
@@ -224,19 +357,30 @@
   describe "comment_reply_sent_notification" do
     subject(:email) { CommentMailer.comment_reply_sent_notification(comment) }
 
-    let(:parent_comment) { create(:comment) }
+    let(:parent_comment) { create(:comment, pseud: commenter_pseud) }
     let(:comment) { create(:comment, commentable: parent_comment) }
 
     it_behaves_like "an email with a valid sender"
+    it_behaves_like "a multipart email"
     it_behaves_like "it retries when the comment doesn't exist"
     it_behaves_like "a notification email with a link to the comment"
     it_behaves_like "a notification email with a link to the comment's thread"
+    it_behaves_like "a notification email with the commenters pseud and username"
+
+    context "when the comment is by an official user using their default pseud" do
```

Comment:
I don't think we need to have it for every test, but I think it would be good to include the registered user with default pseud case somewhere as well

Reply:
Added, was simple enough to make a shared examples and include it everywhere.